.github/workflows/test-and-release.yaml

```diff
@@ -0,0 +1,116 @@
+name: Test and release
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - id: checkout
+        name: Checkout repository
+        uses: actions/checkout@v3
+
+      - id: setup-node
+        name: Setup Node.JS
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: 'yarn'
+
+      - id: install-deps
+        name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - id: lint
+        name: Lint code
+        run: yarn run eslint ./lib/ --ext .js --debug
+
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    needs: lint
+    strategy:
+      matrix:
+        node-version: ['14', '16', '18']
+    steps:
+      - id: checkout
+        name: Checkout repository
+        uses: actions/checkout@v3
+
+      - id: setup-node
+        name: Setup Node.JS
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'yarn'
+
+      - id: install-deps
+        name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - id: build
+        name: Build library
+        run: yarn build
```

Comment:
_Technically_ not needed, b/c we call [build in test](https://github.com/reuters-graphics/bluprint/blob/master/package.json#L48). Benefit of doing those internal build calls in package.json as I see it, btw, is it makes this workflow work as template regardless whether our lib has a build step (many don't these days...).

Reply:
Yes, you are correct. I cut the build step.